common/ntp/ntp.go

```diff
@@ -0,0 +1,35 @@
+package ntp
+
+import (
+	"time"
+
+	beevik_ntp "github.com/beevik/ntp"
+)
+
+const (
+	// tolerance range of local clock with NTP time server
+	toleranceRange = time.Duration(50 * time.Millisecond)
+)
+
+// IsLocalTimeAccurate returns whether the local clock acturate or not
+func IsLocalTimeAccurate(ntpServer string) bool {
+	response, err := beevik_ntp.Query(ntpServer)
```

Comment:
Probably we shall do a sanity check on the ntpServer configuration here. So that if a user has wrong ntpServer config, he will not get a non-sense error saying system clock is off.

Reply:
let me see if we can get more error info output.